src/main.rs

```diff
@@ -136,6 +140,7 @@ enum Command {
         /// The citation keys to delete.
         citation_keys: Vec<RecordId>,
         /// Delete without prompting.
+        /// This is implied by the `--non-interactive` flag.
```

Comment:
I don't know if this is a good idea, perhaps `--non-interactive` should just mean "fail without prompting"?

Reply:
Wait this is inside the `delete` command so it's totally fine. Resolving this now.

Reply:
Actually maybe the help message is a bit long? What does it looks like to run `autobib help delete`?

Reply:
It's a bit long. I'll break it into a new line